packages/app-desktop/services/plugins/PlatformImplementation.ts

```diff
@@ -82,8 +85,34 @@ export default class PlatformImplementation extends BasePlatformImplementation {
 	}
 
 	public get imaging(): ImagingImplementation {
+		const createFromPdf = async (path: string, options: CreateFromPdfOptions) => {
+			const tempDir = join(Setting.value('tempDir'), uuid.createNano());
+			await shim.fsDriver().mkdir(tempDir);
+			try {
+				const paths = await shim.pdfToImages(path, tempDir, options);
+				return paths.map(path => nativeImage.createFromPath(path));
+			} finally {
+				await shim.fsDriver().remove(tempDir);
+			}
+		};
 		return {
-			nativeImage: nativeImage,
+			nativeImage: {
+				async createFromPath(path: string) {
+					if (path.endsWith('.pdf') || path.endsWith('.PDF')) {
```

Comment:
Or `path.toLowerCase().endsWith('.pdf')`? (just to handle that one edge case where the pdf is going to be named .Pdf)